azurerm/internal/services/cosmos/cosmosdb_account_resource_test.go

```diff
@@ -1660,6 +1711,112 @@ resource "azurerm_cosmosdb_account" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomString, data.RandomString, data.RandomInteger, string(kind), string(consistency))
 }
 
+func (CosmosDBAccountResource) basicWithBackupPolicyPeriodic(data acceptance.TestData, kind documentdb.DatabaseAccountKind, consistency documentdb.DefaultConsistencyLevel) string {
```

Comment:
Since the argument `kind`, `consistency` here are always passed in with same value, shall we simply remove these arguments and hardcode the used configs instead?

Reply:
Since `basic` requires `kind` and `consistency`, it could be clearer to have the same argument as the basic?

Reply:
Ah, I see.